whirlpool/backendselector.py

```diff
@@ -27,7 +27,6 @@ class Region(Enum):
     Region.US: {"base_url": "https://api.whrcloud.com"},
 }
 
-
```

Comment:
this was formatted with black, and we should keep it. please also try to use black while adding code to keep it formatted
I plan to add a github job for validation, but have not yet.

Reply:
Not even sure why it was removed... sorry